### DISCUSSION TITLE
Recover account with 2FA and unliked email

### DISCUSSION THREAD

**Discussion by s-homon** [community/community#66984] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I need to restore access to my old GitHub account. The 2FA was enabled for it, but I don't have access to the mobile number, the email was unlinked and used for another account, and recovery codes are lost also. 

I have only a username, password, and ssh key.

Is it possible to do something?

Thanks

**Comment by hafiz-muhammad** [comment]
You can refer to this GitHub Doc: [Recovering your account if you lose your 2FA credentials.](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#request-help-with-two-factor-authentication)

Also remember it's important to backup things like your passwords and recovery codes in more than one place.

**Reply by s-homon** [reply]
I tried, but during recovering github sends email that was unlinked from account, and  the email does not arrive

**Reply by hafiz-muhammad** [reply]
In the documentation it says: 

_**"For security reasons, GitHub Support [will not be able to restore access to accounts](https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy) with two-factor authentication enabled if you lose your two-factor authentication credentials or lose access to your account recovery methods."**_

_**" If you cannot use any recovery methods, you have permanently lost access to your account. However, you can unlink an email address tied to the locked account. The unlinked email address can then be linked to a new or existing account."**_

In the future just remember to backup important things like account passwords and recovery codes in multiple places like different personal machines, storage devices and cloud storage. I do recommend encrypting the storage devices and files before uploading to a cloud storage.
